extensions/window/layout.lua

```diff
@@ -424,8 +424,8 @@ local function performPendingActions()
             win:setFrame(frame)
             command.log.f('rule %s: %s (%d) moved to %s',idx,appname,id,frame.string)
           end
-        elseif action==NOACTION then
```

Comment:
This one looks like it was deliberate to make sure we would do nothing when `action==NOACTION`?